stable/elasticsearch/Chart.yaml

```diff
@@ -3,7 +3,7 @@ appVersion: "1.0"
 description: A Helm chart for Elasticsearch
 name: elasticsearch
 icon: https://www.elastic.co/static/images/elastic-logo-200.png
-version: 1.0.0-elasticsearch6.5.1
+version: 1.1.0-elasticsearch6.5.1
 maintainers:
   - name: FFX Blue Operations
     email: blueops@fairfaxmedia.com.au
```

Comment:
Could you please use the github@fairfaxmedia.com.au account - see the k8s-secret-updater chart

Reply:
Will do.

Reply:
https://github.com/fairfaxmedia/charts/blob/bf5058f62c164f3b4ee19c783230c0b0ff196642/stable/k8s-secret-updater/Chart.yaml#L6-L7

Reply:
Change made.